BedrockCore.cpp

```diff
@@ -28,6 +28,36 @@ class AutoScopeRewrite {
     bool (*_handler)(int, const char*, string&);
 };
 
```

Comment:
Can you add some documentation
`// Return the number of seconds left before this command times out.  Throw an error if has timed out`

Reply:
Added in the .h file.